dev-lua/lua-argparse/lua-argparse-0.6.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Command line argument parser for the Lua Programming Language"
+HOMEPAGE="https://github.com/mpeterv/argparse"
+SRC_URI="https://github.com/mpeterv/argparse/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc luajit test"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )
+	test? ( dev-lua/busted )"
+RDEPEND="
+	>=dev-lang/lua-5.1:=
+	luajit? ( dev-lang/luajit:2 )"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN//lua-/}-${PV}"
+
+src_compile() {
+	if use doc; then
+		sphinx-build docsrc html || die
+		rm -rf "${S}"/html/{.doctrees,_sources}
```

Comment:
please `|| die` here, `-f` still can return non-zero for permission errors and other situations. if possible, avoid `-f` in rm calls.

Reply:
fixed